X16 Reference - Appendix F - 65C816 Processor.md

```diff
@@ -0,0 +1,2470 @@
+
+---
+
+### STX
+
+**Store Index X to Memory**
+
+```text
+SYNTAX         MODE          HEX LEN CYCLES      FLAGS   
+STX $20        dir           86  2   4-x+w       ........ .
+STX $20,Y      dir,Y         96  2   5-x+w       ........ .
+STX $1234      abs           8E  3   5-x         ........ .
+```
+
+Stores the value in .X to a memory address.
+
+When the flag **x** is 0, the value saved will be a 16-bit number, using two
+bytes of memory. When **x** is 1, the value will be an 8-bit number, using one
+byte of RAM.
+
+
+[[Opcodes](#instructions-by-opcode)] [[By Name](#instructions-by-name)] [[By Category](#instructions-by-category)]
+
+---
+
+### STY
+
+**Store Index Y to Memory**
+
+```text
+SYNTAX         MODE          HEX LEN CYCLES      FLAGS   
+STY $20        dir           84  2   4-x+w       ........ .
+STY $20,X      dir,X         94  2   5-x+w       ........ .
+STY $1234      abs           8C  3   5-x         ........ .
+```
+
+Stores the value in .Y to a memory address.
+
+When the flag **x** is 0, the value saved will be a 16-bit number, using two
+bytes of memory. When **x** is 1, the value will be an 8-bit number, using one
+byte of RAM.
+
+
+[[Opcodes](#instructions-by-opcode)] [[By Name](#instructions-by-name)] [[By Category](#instructions-by-category)]
+
+---
+
+### STZ
+
+**Store Sero to Memory**
+
+```text
+SYNTAX         MODE          HEX LEN CYCLES      FLAGS   
+STZ $20        dir           64  2   4-m+w       ........ .
+STZ $20,X      dir,X         74  2   5-m+w       ........ .
+STZ $1234      abs           9C  3   5-m         ........ .
+STZ $1234,X    abs,X         9E  3   6-m         ........ .
+```
+
+Stores a zero to a memory address.
+
+When **m** is 0, the value saved will be a 16-bit number, using two bytes of
+memory. When **m** is 1, the value will be an 8-bit number, using one byte of
+RAM.
+
+
+[[Opcodes](#instructions-by-opcode)] [[By Name](#instructions-by-name)] [[By Category](#instructions-by-category)]
+
+---
+
+### TAX
+
+**Transfer Accumulator to Index X**
+
+```text
+SYNTAX         MODE          HEX LEN CYCLES      FLAGS   
+TAX            imp           AA  1   2           n.....z. .
+```
+
+Copies the contents of .A to .X.
+
+The transfer (copy) is 16 bits only if **m** and **x** are both 0.
+
+If **m** or **x** is 1, the copy will only be 8 bits, and the upper byte of .X
+will nt be affected.
+
+
+[[Opcodes](#instructions-by-opcode)] [[By Name](#instructions-by-name)] [[By Category](#instructions-by-category)]
+
+---
+
+### TAY
+
+**Transfer Accumulator to Index X**
+
+```text
+SYNTAX         MODE          HEX LEN CYCLES      FLAGS   
+TAY            imp           A8  1   2           n.....z. .
+```
+
+Copies the contents of .A to .Y.
+
+The transfer (copy) is 16 bits only if **m** and **x** are both 0.
+
+If **m** or **x** is 1, the copy will only be 8 bits, and the upper byte of .Y
+will nt be affected.
```

Comment:
"nt" typo.  

Also, this is not entirely true.

![image](https://github.com/X16Community/x16-docs/assets/395186/0357765a-e1e4-4628-b964-86084047133f)